### ISSUE TITLE
"colima start" does not start, "invalid networkDriver:"

### ISSUE THREAD

**Issue by RobertoMD** [abiosoft/colima#604]
### Description

After upgrading my mac to Ventura, and reinstalling the latest version of colima, I get this message when trying to start colima:
> colima start
FATA[0000] error in config: invalid networkDriver: ''

If I create a new configuration ("debug"), it works perfectly
> colima start debug
INFO[0000] starting colima [profile=debug]
INFO[0000] runtime: docker
INFO[0000] preparing network ...                         context=vm
INFO[0000] starting ...                                  context=vm
INFO[0021] provisioning ...                              context=docker
INFO[0021] starting ...                                  context=docker
INFO[0027] done

, so there must be something in the default configuration that can be reset, but can't get what.

### Version

Colima Version:  HEAD-d2b24d2
Lima Version: 0.14.2
Qemu Version: 7.2.0

### Operating System

- [ ] macOS Intel <= 12 (Monterrey)
- [X] macOS Intel >= 13 (Ventura)
- [ ] macOS M1 <= 12 (Monterrey)
- [ ] macOS M1 >= 13 (Ventura)
- [ ] Linux

### Output of `colima status`

> colima status
FATA[0000] colima is not running
> colima status debug
INFO[0000] colima [profile=debug] is running using QEMU
INFO[0000] arch: aarch64
INFO[0000] runtime: docker
INFO[0000] mountType: sshfs
INFO[0000] socket: unix:///Users/roberto/.colima/debug/docker.sock

### Reproduction Steps

1. colima start

### Expected behaviour

I'd expect that colima start starts the default configuration instead of throwing an error "invalid networkDriver:"

**Comment by abiosoft** [comment]
Thanks for reporting.

This is most likely due to some recently added input validations.

**Comment by jaepetto** [comment]
+1

**Comment by ricmano** [comment]
+1

**Comment by rpkoller** [comment]
ran into the same after updating to colima 0.5.3 on first `colima start` ( with lima 0.15.0 on macos 12.6.1)

**Comment by abiosoft** [comment]
@rpkoller 

> ran into the same after updating to colima 0.5.3 on first colima start ( with lima 0.15.0 on macos 12.6.1)

An hotfix release is on the way for this. It was not covered in the 0.5.3 as assumed.

> i've manually entered slirp instead since colima is using lima and lima qemu. with slirp in place i was able to start colima again. is that correct or would `gvproxy`still be the preferable choice? 

Either of them works fine for 99% of users. There are some edge cases that each driver handles better than the other.
You likely would not notice any difference irrespective of the driver you use.

**Comment by rpkoller** [comment]
thank you for the explanation and the hotfix. 

> Either of them works fine for 99% of users. There are some edge cases that each driver handles better than the other. You likely would not notice any difference irrespective of the driver you use.

and yes i can confirm `slirp`as well as `gvproxy` work well for me. i've just picked `slirp` in the first place because the code comment mentioned qemu which is used by lima and therefore colima. i should have started thinking cuz i was editing a colima config file which recommends `gvproxy` as the default ^^ but with the invalid network driver error i was a little panicking and not rationally thinking ^^ but both working fine. so thanks again for the fast fix!

**Comment by RobertoMD** [comment]
> thank you for the explanation and the hotfix.
> 
> > Either of them works fine for 99% of users. There are some edge cases that each driver handles better than the other. You likely would not notice any difference irrespective of the driver you use.
> 
> and yes i can confirm `slirp`as well as `gvproxy` work well for me. i've just picked `slirp` in the first place because the code comment mentioned qemu which is used by lima and therefore colima. i should have started thinking cuz i was editing a colima config file which recommends `gvproxy` as the default ^^ but with the invalid network driver error i was a little panicking and not rationally thinking ^^ but both working fine. so thanks again for the fast fix!

Hi rpkoller, could you please post the contents of your `colima.yaml `file? Mine does not have a "driver" entry. I tried adding it manually (`driver: slirp` , or `driver: gvproxy`) but I'm still getting the same error message:
`FATA[0000] error in config: invalid networkDriver: ''`

**Comment by abiosoft** [comment]
@RobertoMD this should be fixed already if you upgrade to `0.5.4`. 

Nonetheless, the driver is under network section. Or you can use the `--network-driver` flag.
```yaml
# Network configurations for the virtual machine.
network:
   ...
  # Network driver to use (slirp, gvproxy), (requires vmType `qemu`)
  #   - slirp is the default user mode networking provided by Qemu
  #   - gvproxy is an alternative to VPNKit based on gVisor https://github.com/containers/gvisor-tap-vsock
  # Default: gvproxy
  driver: gvproxy
```

**Comment by RobertoMD** [comment]
Thanks, I've upgraded to HEAD version and it's working now.
